Documentation/compatibility/unexpected-invalidcastexception-from-invokemethod-activity-in-wf4.md

```diff
@@ -13,16 +13,23 @@ Minor
 Planned
 
 ### Change Description
-Using a System.Activities.Statements.InvokeMethod that targets a method with a nullable parameter can throw an InvalidCastException.
+
+Using a
+<xref:System.Activities.Statements.InvokeMethod> that
+targets a method with a nullable parameter can throw an
+<xref:System.InvalidCastException?displayProperty=name>.
 
 - [ ] Quirked
 - [ ] Build-time break
 
 ### Recommended Action
-This behavior was reverted in a .NET Framework 4.5 servicing release. Please update the .NET Framework 4.5 (or upgrade to .NET Framework 4.5.1 or later) to fix the issue.
+
+This behavior was reverted in a .NET Framework 4.5 servicing release. Please
+update the .NET Framework 4.5 (or upgrade to .NET Framework 4.5.1 or later) to
+fix the issue.
 
 ### Affected APIs
-* `P:System.Activities.Statements.InvokeMethod.Parameters`
+* `P:<xref:System.Activities.Statements.InvokeMethod>.Parameters`
```

Comment:
This won't resolve to a valid URL target. It should be <xref:System.Activities.Statements.InvokeMethod.Parameters?displayProperty=fullName>
In general, affected APIs should all be fully qualified.